tests/test_simulation.py

```diff
@@ -1187,13 +1257,20 @@ def pos_factor(qid):
         np.testing.assert_allclose(
             rydberg_samples[qid]["phase"][time_slice], pulse1.phase
         )
+    with pytest.warns(
+        DeprecationWarning, match="The `Simulation` class is deprecated"
+    ):
+        _sim = Simulation(seq, with_modulation=True, config=sim_config)
+        _sim.run()
```

Comment:
Why the `run()`?

Reply:
I needed a call to a method/attribute of QutipEmulator that is not in Simulation to reach 100% coverage (call to setattr)

Reply:
Ok, I see

Reply:
Maybe leave a comment explaining the intent then

Reply:
It is deleted I have found another way to solve this issue.